modules/logstash/data/config.tpl.conf

```diff
@@ -42,14 +42,14 @@ filter {
     if [info] {
         mutate {
             rename => {
-                "info" => "%{[type]}_info"
```

Comment:
What requires the double `%%`? Is this to tell Terraform to leave it alone?

Reply:
Yes, exactly